src/pages/careers/locations/_location-layout.js

```diff
@@ -166,12 +169,12 @@ export const LocationLayout = ({ location, images }) => {
                     </Flex>
                 </FirstSection>
             </Container>
-            <SectionContainer padding={location.has_iframe ? '0rem' : '0 0 12rem'}>
+            <SectionContainer padding="0">
                 <Header
                     align="center"
                     as="h2"
-                    size={'var(--text-size-header-1)'}
-                    style={{ marginBottom: '6.4rem' }}
+                    size={'var(--text-size-header-5)'}
```

Comment:
```suggestion
                    size="var(--text-size-header-5)"
```
I guess it can be used as string. Please check 🙏🏻